CHANGELOG.rst

```diff
@@ -21,6 +21,9 @@ Added
 - Add to_jwk static method to ECAlgorithm by @leonsmith in https://github.com/jpadilla/pyjwt/pull/732
 - Add ``get_algorithm_by_name`` as a method of ``PyJWS`` objects, and expose
   the global PyJWS method as part of the public API
+- Add ``compute_hash_digest`` as a method of ``Algorithm`` objects, which uses
```

Comment:
@sirosen could you update the changelog?

Reply:
Yes, I'll do a quick rebase to cleanup the conflict and move this to the latest section.